### ISSUE TITLE
what's diffrent from jquery with this

### ISSUE THREAD

**Issue by xulayen** [cheeriojs/cheerio#1216]
what's diffrent from jquery with this?

**Comment by Kotsuha** [comment]
I'm new to Node.js so I might not be completely correct. jQuery is a JavaScript library. Traditionally JavaScript runs only on the client side. Node.js enables JavaScript to be used for server-side scripting. Since the environment is different, jQuery cannot be used with Node.js ([at least not directly](https://stackoverflow.com/a/8014561/6378871)). If you have the need to traverse/manipulate with DOM, there are several options and Cheerio is one of them. This is the best I can give and you can search for the rest.

TLDR: Think of cheerio as (a subset of) jQuery for Node.js

**Comment by fb55** [comment]
@Kotsuha All of this is correct, thanks for chiming in!
